Update Logos to use a white color scheme

escherlies opened the pull request:
To make every logo and sponsor equal as well as ensuring consistency across our design, let's update the color schemes of the logos to white.

<img width="2304" height="4096" alt="Screenshot From 2025-08-20 12-45-19" src="https://github.com/user-attachments/assets/8437cbc0-47e4-4a80-95ef-ee460e6103e8" />

escherlies committed:
Update SVG assets to use a white color scheme

This change updates the color scheme of various SVG assets to white. The update aims to ensure consistency across the application's visual elements and improve the overall aesthetic by aligning with the current design guidelines.

a-kenji commented:
Unsure that we can substantially alter the logos of sponsors, see https://github.com/nixcon/2025.nixcon.org/pull/91#discussion_r2277533837.

But in general I am for it and approve it. 
But want to have @infinisil look over it first.

Also some logos substantially degrade this way, for example determinate systems and helsinki.

escherlies commented:
Thanks @a-kenji , addressing your concerns and clarifying my arguments:

- I think https://github.com/nixcon/2025.nixcon.org/pull/91#discussion_r2277533837 is not applicable here since it altered the colors inherently whereas black and white single-color logo versions are common practice
- It is common business practice that logos can be displayed as a single-color logo (for print media for example!)
- I think it's just **fair** that every logo is the same color since it eliminates hierarchy because some logos are less visible and other logos are more visible
  - Many logos gain visibility i.e. express vpn, pareto, PDT partners etc. which weren't really visible before
  - Determinate systems still look really great and so do helsinki systems 


What do you think @infinisil ?

escherlies commented:
Adding some examples to visualize the improved readability


---

<img width="250" height="178" alt="Screenshot From 2025-08-20 14-12-57" src="https://github.com/user-attachments/assets/ab49b126-7f24-42f0-8dc9-f0ca9932cbff" />
<img width="250" height="178" alt="Screenshot From 2025-08-20 14-16-10-1" src="https://github.com/user-attachments/assets/e598a5d8-99c2-43fc-9c2f-63db90b0b832" />

---

<img width="318" height="202" alt="Screenshot From 2025-08-20 14-19-21" src="https://github.com/user-attachments/assets/0447f912-b4e1-4cef-b790-7b439463b369" />
<img width="318" height="202" alt="Screenshot From 2025-08-20 14-19-27" src="https://github.com/user-attachments/assets/54feac60-74ef-4fea-83da-3da70b85b536" />

---

<img width="302" height="176" alt="Screenshot From 2025-08-20 14-19-46" src="https://github.com/user-attachments/assets/9262dc4b-9f63-46ab-8965-2a7d6c2310b3" />
<img width="308" height="172" alt="Screenshot From 2025-08-20 14-20-06" src="https://github.com/user-attachments/assets/6b7cd78e-b6a1-4414-adb2-7623e318930b" />

---

<img width="318" height="190" alt="Screenshot From 2025-08-20 14-18-49" src="https://github.com/user-attachments/assets/07fab857-2ff1-4720-9c60-e3e0d289e4c7" />
<img width="318" height="190" alt="Screenshot From 2025-08-20 14-18-54" src="https://github.com/user-attachments/assets/6ba99638-89d0-4cd0-a153-9763fbfce526" />

---

<img width="274" height="170" alt="Screenshot From 2025-08-20 14-23-33" src="https://github.com/user-attachments/assets/44fc9044-d5cb-4f77-b9bf-96b9e2fa51a9" />

<img width="268" height="154" alt="Screenshot From 2025-08-20 14-23-45" src="https://github.com/user-attachments/assets/a83cd974-d478-4d8b-870f-df8aa54e7ff6" />

infinisil commented:
I think this is fine :)
